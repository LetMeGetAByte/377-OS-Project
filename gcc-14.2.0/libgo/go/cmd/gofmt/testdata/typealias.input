version https://git-lfs.github.com/spec/v1
oid sha256:575242d5a3fdd7c43455c9eb5b63899dca66fcb15b49114d3d67008e130ae78e
size 360
