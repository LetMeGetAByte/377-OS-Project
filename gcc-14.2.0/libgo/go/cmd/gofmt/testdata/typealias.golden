version https://git-lfs.github.com/spec/v1
oid sha256:77663d7e3fa4f93ec86ee32b59c6c31781b3622b577fc18127af38768e961c11
size 365
